jbpm-console-ng-showcase/src/main/java/org/jbpm/console/ng/server/impl/DashbuilderBootstrap.java

```diff
@@ -106,10 +111,10 @@ protected void registerDataSetDefinitions() {
                 .uuid(HUMAN_TASKS_WITH_USER_DATASET)
                 .name("Human tasks and users")
                 .dataSource(JBPM_DATASOURCE)
-                .dbSQL("select  t.activationTime, t.actualOwner, t.createdBy, "
```

Comment:
Not sure if it is desired to change  the column names to all-lowercase. I believe this query has been rewritten to CamelCase recently to avoid case sensitivity issues with MySQL DB